sequins_test.go

```diff
@@ -63,6 +61,108 @@ func TestSequins(t *testing.T) {
 func TestSequinsNoValidDirectories(t *testing.T) {
 	backend := backend.NewLocalBackend("test_data/0")
 	s := newSequins(backend, sequinsOptions{"test_data/0", false})
-	err := s.start("localhost:0")
+	err := s.init()
 	assert.Error(t, err)
 }
+
+// make sure that reads that occur during an update DTRT
+func TestSequinsThreadsafe(t *testing.T) {
+	assert.NoError(t, os.RemoveAll("test_scratch"))
```

Comment:
it'd be a little nicer to grab a tmp directory, I think. (the s3 and hdfs tests do, for example).